### DISCUSSION TITLE
Planetscale x Prisma (multiple databases)

### DISCUSSION THREAD

**Discussion by jlorezz** [prisma/prisma#9960] [Q&A]
Has there been anything been done with planet-scale x Prisma for multiple databases? 
we would need a total of 3 databases
(production, uat, and development)
is there a guide to this?

**Comment by austincrim** [comment]
Hey @jlorezz 👋,

We don't have a guide for this right now, but we are planning to create one soon. It would be awesome if you could [create an issue](https://github.com/prisma/docs/issues/new?assignees=&labels=docs&template=doc_request.md) in our docs repo outlining what specific questions you would like us to address in a future guide!

In the meantime, [this community-written article](https://davidparks.dev/blog/planetscale-deployment-with-prisma/) would be a great place to start.

We are also building a solution to support multiple environments and we will be discussing it more at our Serverless Data Conference on November 18th. Be on the lookout for more information about the conference coming soon 😃.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
